sources/healthdog/src/main.rs

```diff
@@ -0,0 +1,138 @@
+/*!
+# Introduction
+
+Healthdog sends anonymous information about the health of a Bottlerocket host.
+It does so by sending key-value pairs as query params in an HTTP GET request.
+
+Healthdog also has the ability to check that a list of critical services is running.
+It does so using `systemctl` and reports services that are not healthy.
+
+# What it Sends
+
+### The standard set of metrics:
+
+* `sender`: the application sending the report.
+* `event`: the event that invoked the report.
+* `version`: the Bottlerocket version.
+* `variant`: the Bottlerocket variant.
+* `arch`: the machine architecture, e.g.'x86_64' or 'arm'.
+* `region`: the region the machine is running in.
+* `seed`: the seed value used to roll-out updates.
+* `version-lock`: the optional setting that locks Bottlerocket to a certain version.
```

Comment:
```suggestion
* `version-lock`: the optional setting that controls Bottlerocket update selection.
```
Not necessarily a certain version, it can be "latest" (and maybe other things in the future)

Reply:
Fixed in [2a5aef0](https://github.com/bottlerocket-os/bottlerocket/compare/f515d383c9e181c8a531a02414b320ff9231a81f..2a5aef04d7abcae5c91bf8cec84b8e95c7ccd306)